nvme-print-json.c

```diff
@@ -4787,3 +4786,34 @@ struct print_ops *nvme_get_json_print_ops(nvme_print_flags_t flags)
 	json_print_ops.flags = flags;
 	return &json_print_ops;
 }
+
+void obj_add_byte_array(struct json_object *o, const char *k, unsigned char *buf, int len)
+{
+	int i;
+
+	_cleanup_free_ char *value = NULL;
+
+	if (!o || !k || !buf || !len)
```

Comment:
Is it necessary to handle all these cases? I don't see the other JSON value add functions checking their arguments like this. Seems like this would be papering over a bug in the caller...

Reply:
Which do you suggest?
1. Ignore the invalid value with the checking then the function caused a null pointer exception or any unexpected error. (I do not think this function behavior correct.)
2. Change the void function to return any error by the checking?